mergify_engine/clients/github.py

```diff
@@ -112,6 +113,17 @@ def items(self, url, api_version=None, list_items=None, **params):
             else:
                 break
 
+    def request(self, *args, **kwargs):
+        try:
+            return super().request(*args, **kwargs)
+        finally:
+            self._nb_requests += 1
+
+    def close(self):
+        super().close()
+        statsd.histogram("engine.github_session.size", self._nb_requests)
+        statsd.increment("engine.github_request.count", self._nb_requests)
```

Comment:
I wonder if we should prefix with `engine` here: if we either track them from the dashboard at some point, it'd be nice to be allowed to group them by app.
I think `github.http_session.size` and `github.http_request.size` maybe better names.

Reply:
Work for me.